app/models/legislator.rb

```diff
@@ -39,7 +39,9 @@ class Legislator < ActiveRecord::Base
   has_many :campaigns, through: :targets
   has_many :active_campaigns, -> { merge(Campaign.active) }, through: :targets, source: :campaign
   has_many :sponsorships
+  has_many :current_sponsorships, -> { current }, class_name: "Sponsorship"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.